xcell/cls/cl.py

```diff
@@ -226,12 +226,32 @@ def get_cl_file(self):
                 cl -= nl
             # Note that while we have subtracted the noise
             # bias from `cl_cp`, `cl_cov_cp` still includes it.
+            correction = None
```

Comment:
I think this was only needed for the ACT kappa map, right? Could we remove these lines from this PR to keep it simple?